lib/presentation/organisms/intro/tips_organism.dart

```diff
@@ -37,6 +40,15 @@ class TipsOrganism extends StatelessWidget {
             translate: false,
           ),
           const SeparatorAtom(variant: SeparatorVariant.farApart),
+          Center(
+            child: Platform.isAndroid
+                ? ButtonAtom(
+                    variant: ButtonVariant.primary,
+                    onPressed: () => DndController.instance.toggleDnd(),
```

Comment:
I think it can be written like this

` onPressed: DndController.instance.toggleDnd,`

Reply:
This will not work because toggleDnd is a Future<void> function and the onPressed parameter only accepts void Functions, I already tried it without () =>

Reply:
Worked on my machine 😉, try again by pasting my row